pages/en/billing.mdx

```diff
@@ -2,75 +2,74 @@
 
-Before making queries on The Graph Network, you will need to add GRT to your account. This will take about 10-30 minutes depending on network traffic. To add GRT to your account, take the following steps:
+6. Click 'Add GRT to billing balance' to add the GRT to your account. Sign the associated transaction in your wallet. This will cost gas.
 
-1. Purchase GRT and ETH on an exchange or decentralized exchange of your choice
-2. Send the GRT and ETH to your wallet
-3. Bridge GRT to Polygon using the Subgraph Studio UI
+7. Once the transaction is confirmed, you'll see the GRT added to your account within an hour.
 
-   a) You will receive 0.001 Matic within a few minutes of sending GRT to the Polygon bridge. You can monitor the transaction by inputting your wallet address into the search bar on [Polygonscan](https://polygonscan.com/).
+### Withdrawing GRT using a crypto wallet
 
-4. Add bridged GRT to the billing contract on Polygon in the Subgraph Studio UI. The billing contract address is: [0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE](https://polygonscan.com/address/0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE).
+> This section is written assuming you have deposited GRT into your billing account on [Subgraph Studio](https://staging.thegraph.com/studio/billing/) and that you're on the Arbitrum network.
 
-   a) In order to complete step #4, you must switch from Ethereum to the Polygon network. You can do this using the Subgraph Studio.
+1. Go to the [Subgraph Studio Billing page](https://staging.thegraph.com/studio/billing/).
 
-If you used your API keys over the course of any given week, you will receive an invoice based on the query fees you have generated during this period. This invoice will be paid using GRT available in your balance. Query volume is evaluated by the API keys you own. Your balance will be updated after fees are withdrawn.
+2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect".
 
-### Understanding Invoices in Subgraph Studio
+3. Click the dropdown next to the 'Add GRT' button at the center of the page. Select withdraw GRT. A side panel will appear.
 
-When using The Graph Network, dapps and users pay for query fees in GRT, the protocol's native work-utility token. Maintaining a positive balance for an account is the user's or dapp's responsibility.
+4. Enter the amount of GRT you would like to withdraw.
 
-Here are ways to be notified about and/or check a query fee balance:
+5. Click 'Withdraw GRT' to withdraw the GRT from your account. Sign the associated transaction in your wallet. This will cost gas. The GRT will be sent to your Arbitrum wallet.
 
-- You can set up a billing notification that will be sent to an email address [here](https://thegraph.com/studio/settings/).
+6. Once the transaction is confirmed, you'll see the GRT withdrawn from your account in your Arbitrum wallet within an hour.
```

Comment:
Could probably remove the 'within an hour' part. Since everything is taking place on arbitrum for Withdraw (unlike in the 'Add' step) the user will see the funds in their wallet immediately after the transaction is successful.

Reply:
Awesome! Didn't know that - thank you!